src/Analysis/Ast/Test/AnalysisTestBase.cs

```diff
@@ -38,7 +38,9 @@
 
 namespace Microsoft.Python.Analysis.Tests {
     public abstract class AnalysisTestBase {
-        protected TimeSpan AnalysisTimeout { get; set; } = TimeSpan.FromMinutes(1);
+        protected const int AnalysisTimeoutInMS = 1000 * 60;
```

Comment:
Any reason for this change? Seems unrelated (but has the same affect).

Reply:
I need the const for attribute. so made the const and made existing one to consume that const

Reply:
Oh, oops. My github diff had the uses of this new constant hidden so I missed it.